packages/components/src/Drawer/Drawer.component.js

```diff
@@ -107,7 +107,7 @@ export function subtitleComponent(subtitle) {
 }
 
 function DrawerTitle({ title, subtitle, children, onCancelAction, getComponent }) {
-	if (!title) {
+	if (!title && !children) {
```

Comment:
this change do the exact inverse of the PR description.

Reply:
no it's not, is return null only on a more specific condition which comply with description

Reply:
yep sorry, my bad